CODE-OF-CONDUCT.md

```diff
@@ -1,46 +1,135 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
```

Comment:
The link in this badge points to this same file. Is the link supposed to point to something on contributor covenant, or is the badge supposed to be in a different file?

Reply:
I just saw your comment, this badge should be pointing to [contributor covenant's code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html) only.